airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/WorkspacePersistenceTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.persistence;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.lang.MoreBooleans;
+import io.airbyte.config.Geography;
+import io.airbyte.config.SourceConnection;
+import io.airbyte.config.StandardSync;
+import io.airbyte.config.StandardWorkspace;
+import io.airbyte.db.ExceptionWrappingDatabase;
+import io.airbyte.validation.json.JsonValidationException;
+import java.io.IOException;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+@SuppressWarnings({"PMD.LongVariable", "PMD.AvoidInstantiatingObjectsInLoops"})
+class WorkspacePersistenceTest extends BaseConfigDatabaseTest {
```

Comment:
Changes in this file:
* lift and shift of tests from `ConfigRepository`.
* replace `ConfigPersistence` with `ConfigRepository`.
* intentionally not changing any of the tests themselves.